docs/developer/general/json_web_tokens.rst

```diff
@@ -35,28 +34,15 @@ Relevant documentation
   services. Auth0 is a certified OIDC provider and their documentation is more
   accessible than the OIDC specification.
 
-Introduction
-------------
 
-In the scope of the platform, JWTs are used as bearers of proof. In simple
-terms, a valid token holds the proof of the claims it contains.
+Usage in Renku
+--------------
 
-`Access Tokens <https://tools.ietf.org/html/rfc6749#section-1.4>`_ from the
-identity manager (keycloak) are used to prove:
+Keycloak serves as an identity provider to GitLab. Personal `access
```

Comment:
I would say that it serves as an ID provider to *renku* not just gitlab (although I realize gitlab is the only place it's used at the moment)